### DISCUSSION TITLE
problem opening the game

### DISCUSSION THREAD

**Discussion by zachariascrosstales** [Monika-After-Story/MonikaModDev#8534] [general - submod / external content]
ok the problem is that I try to install a mod for the text to speech but I get this message
![2021-12-29](https://user-images.githubusercontent.com/75182844/147696769-b0e7a0a5-86b6-42e6-9b02-17c77dd17e1b.png)

**Comment by ThePotatoGuy** [comment]
Please report issues with submods to the submod's creator. OR make it a discussion post here.
